### ISSUE TITLE
Segmentation fault

### ISSUE THREAD

**Issue by NytePlus** [ShenhanQian/VHAP#35]
I run flame_viewer and flame_editor, both appear segmentation fault.
maybe it's my environment's reason. I use CUDA 11.7 and pytorch 2.0.1+cu117
```
(VHAP) wcc@xc:~/VHAP$ python vhap/flame_viewer.py --param_path /data1/wcc/VHAP/output/1/2025-03-28_14-44-18/tracked_flame_params_30.npz --tex_path /data1/wcc/VHAP/output/1/2025-03-28_14-44-18/eval_30/mesh/frame_00000.jpg
Segmentation fault (core dumped)
(VHAP) wcc@xc:~/VHAP$ python vhap/flame_editor.py
Segmentation fault (core dumped)
```

**Comment by ShenhanQian** [comment]
You can test by running the demo of DeaPyGUI
```
python -m dearpygui.demo
```

**Comment by NytePlus** [comment]
Thanks, dearpygui.demo runs normally. What would be the problem?

**Comment by ShenhanQian** [comment]
It's hard to say. Can you successfully run tracking with VHAP? And can you run the local viewer of GaussianAvatars with the demo result?

**Comment by NytePlus** [comment]
Yes, I can run VHAP tracking and GaussianAvatars demo(including graphical interface).

**Comment by NytePlus** [comment]
Thanks for your answering, it is not a big problem. Can I ask again if this can be trained using my own multi perspective images and camera parameters? I modified the code to train different perspectives of the same timestep together, but the loss cannot be reduced (I used an OpenGL camera with raduis=1)

**Comment by ShenhanQian** [comment]
Yes, you can see examples in other issues that people have successfully applied VHAP to their own data. You can look at the saved visualization to debug. Comparing your results with our example data can be helpful.
